test/Cargo.toml

```diff
@@ -13,3 +13,4 @@ serde_json = "1.0"
 ckb-mock-tx-types = "0.4.0"
 blake2b-rs = "0.1.5"
 faster-hex = "0.6"
+ckb-hash = "0.104.0"
```

Comment:
Why not use 0.118

Reply:
There will be another PR to update these dependencies. There are some minor changes to the project.

Reply:
Updated